packages/navigation/src/history.js

```diff
@@ -6,4 +6,13 @@ import { createHashHistory } from 'history';
 
 // See https://github.com/ReactTraining/react-router/blob/master/FAQ.md#how-do-i-access-the-history-object-outside-of-components
 
-export default createHashHistory();
+let _history;
+
+function getHistory() {
+	if ( ! _history ) {
+		_history = createHashHistory();
+	}
+	return _history;
+}
+
+export { getHistory };
```

Comment:
Just curious here. Why wrap the export in an new object? When this gets imported `import { getHistory } from './history';` I would think that would break (it doesn't).

Does webpack do something here, or am I just looking at it wrong?

Reply:
Since we use `getHistory` as an import and also export it, I used the named export notation here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export#Using_named_exports